frontend/src/metabase/sharing/components/SharingMenu/test/QuestionSharingMenu.enterprise.unit.spec.tsx

```diff
@@ -60,36 +50,37 @@ describe("QuestionSharingMenu > Enterprise", () => {
       );
     });
 
-    it("should show a 'ask your admin to create a public link' tooltip if public sharing is disabled", async () => {
+    it("should show a disabled 'ask your admin to create a public link' menu item if public sharing is disabled", async () => {
       setupQuestionSharingMenu({
         isPublicSharingEnabled: false,
         hasPublicLink: true,
         canManageSubscriptions: false,
         isEnterprise: true,
       });
-      const sharingButton = screen.getByTestId("sharing-menu-button");
+      await openMenu();
 
-      expect(sharingButton).toBeDisabled();
-      expect(sharingButton).toHaveAttribute(
-        "aria-label",
-        "Ask your admin to create a public link",
-      );
+      expect(screen.queryByText("Create alert")).not.toBeInTheDocument();
+      expect(
+        screen.getByRole("menuitem", {
+          name: "Ask your admin to create a public link",
+        }),
+      ).toBeDisabled();
     });
 
-    it("should show a 'ask your admin to create a public link' menu item if public sharing is enabled, but there is no existing public link", async () => {
+    it("should show a disabled 'ask your admin to create a public link' menu item if public sharing is enabled, but there is no existing public link", async () => {
       setupQuestionSharingMenu({
-        isAdmin: false,
```

Comment:
remove this since it's already a default value and other test cases doesn't include this.